arch/riscv64/mcount-support.c

```diff
@@ -0,0 +1,75 @@
+#include <stdlib.h>
+
+#include "libmcount/internal.h"
+#include "utils/filter.h"
+#include "utils/utils.h"
+
+static int mcount_get_register_arg(struct mcount_arg_context *ctx, struct uftrace_arg_spec *spec)
+{
+	/*
+	* TODO:
+	*   This was implemented as a meaningless value because we focused on
+	*   successful builds with no compilation errors.
+	*
+	*   Therefore, it should be modified for future RISC-V 64-bit
+	*   architectures.
+	*/
+}
+static void mcount_get_stack_arg(struct mcount_arg_context *ctx, struct uftrace_arg_spec *spec)
+{
+	/*
+	* TODO:
+	*   This was implemented as a meaningless value because we focused on
+	*   successful builds with no compilation errors.
+	*
+	*   Therefore, it should be modified for future RISC-V 64-bit
+	*   architectures.
+	*/
```

Comment:
Please fix the alignment for the comment.
Before:
```c
    /*
    * TODO: ...
    */
```
After:
```c
    /*
     * TODO: ...
     */
```
In addition, I think the same comment doesn't have to be repeated multiple times. You can just make it shorter as follows.
```c
    /* TODO: not implemented yet. */
```
Anyone can easily understand the meaning.